data/en/lms_2.json

```diff
@@ -27,6 +27,35 @@
         {
             "name": "country",
             "validator": "string"
+        },
+        {
+            "name": "address_line1",
```

Comment:
Im confused as to why we are defining these in metadata if we are not using them?